src/andromede/simulation/xpansion.py

```diff
@@ -155,6 +155,7 @@ def run(
             # TODO Maybe a more robust check and/or return value?
             # For now, it won't look anywhere else because a new
             # architecture should be discussed
+            print(root_dir + "/bin/benders executable not found. Returning True")
```

Comment:
In the future you can use f-strings `f"{root_dir}/bin/benders executable ..."` that is easier to use and more readable when you have more complex strings

Reply:
Indeed a f-string would be nicer. Since It was a simple string, I didn't bother using one but you are right :)

Reply:
Done